### ISSUE TITLE
Logout on frontend redirect to dashboard

### ISSUE THREAD

**Issue by gcrofils** [owen2345/camaleon-cms#242]
On a fresh new install, only the admin user.
Logout has no effect.
it will redirect to dashboard (or loop to dashboard).

video here:

[![2015 12 7-cameleon-logout-01](https://cloud.githubusercontent.com/assets/93443/11619018/98e95b84-9cdd-11e5-80d3-e78ce15daf1f.png)](http://bugs.siz.yt/2015.12.7-cameleon-logout-01.mov)

**Comment by owen2345** [comment]
Hey @gcrofils , Please try again using:
```
gem "camaleon_cms", github: 'owen2345/camaleon-cms'
```
Let me know if it works.

Thank you for the report.

**Comment by gcrofils** [comment]
I changed the Gemfile + Bundle
1) on the current session, it didnt change anything. Still looping (HTTP 302) on the dashboard. 
![capture d ecran 2015-12-08 a 08 42 56](https://cloud.githubusercontent.com/assets/93443/11644526/d049651a-9d87-11e5-91ca-3eee368a99ab.png)

2) I could not reproduce the bug on incognito mode or after deleting all cookies.

3) I switched back to the gem.
I can not reproduce this bug.
